python/sdist/amici/jax/petab.py

```diff
@@ -709,15 +780,70 @@ def _eval_nn(self, output_par: str):
             .to_dict()
         )
 
+        condition_parameter_map = (
+            dict(
+                [
+                    (
+                        petab_id,
+                        self._petab_problem.parameter_df.loc[
+                            self._petab_problem.condition_df.loc[
+                                condition_id, petab_id
+                            ],
+                            petab.NOMINAL_VALUE,
+                        ],
+                    )
+                    if "input"
```

Comment:
this check seems a bit too unspecific. I think you want to construct a sequence of petab id's that are mapped to `$nnId.inputs{[$inputArgumentIndex]{[$inputIndex]}}`?